application/cmd/insgocc/insgocc.go

```diff
@@ -367,13 +322,17 @@ func main() {
 
 			// write include contract + write initialization function
 			initializeOutput := newOutputFlag("")
-			err = openDefaultInitializationPath(initializeOutput)
+			err = openDefaultInitializationPath(initializeOutput, buildInPath)
 			checkError(err)
 
 			err = preprocessor.GenerateInitializationList(initializeOutput.writer, contractList)
 			checkError(err)
 		},
 	}
+	cmdGenerateBuiltins.Flags().StringVarP(
```

Comment:
I don't get idea behind this `importPath`. Why it's so verbose and why it has different format from `contractsPath`? Also it's not a real "path", it's kinda subpath in GOPATH and I don't understand why we should pass two such similar path-like parameters in CLI